microsoft/testsuites/vm_extensions/azure_disk_encryption.py

```diff
@@ -176,6 +176,7 @@ def _enable_ade_extension(self, node: Node, log: Logger, result: TestResult) ->
             )
             vault_name = f"lisa-ade-{random_str}"
 
+        log.info("Key Vault name: %s", vault_name)
```

Comment:
it already has the log containing key vault name `https://github.com/microsoft/lisa/blob/796d49f1f86f9f93d10b6758652a1704c27e804d/microsoft/testsuites/vm_extensions/azure_keyvault_vm_extension.py#L159`

Reply:
there are some tips for writing python code.
e.g. https://mslisa.readthedocs.io/en/main/write_test/guidelines.html#tips-for-writing-code